### ISSUE TITLE
org to pdf conversion with math entity \sigma

### ISSUE THREAD

**Issue by ag91** [jgm/pandoc#2866]
Hi there,

I am failing to convert an org file with math symbols in a pdf file.
I am using

pandoc 1.17.0.2
Compiled with texmath 0.8.5, highlighting-kate 0.6.2.

When I try to convert an org file containing just

`\sigma`

This fails with pdflatex, for the unicode symbol σ:

> ! Package inputenc Error: Unicode char σ (U+3C3)
> (inputenc)                not set up for use with LaTeX.
> 
> See the inputenc package documentation for explanation.
> Type  H <return>  for immediate help.
> ...                                              
>                                                   
> l.53 \emph{σ}
> 
> Try running pandoc with --latex-engine=xelatex.
> pandoc: Error producing PDF

(with the suggested option I get an empty document)
Instead of `\emph{σ}` would not be better the standard latex \sigma?

The latex produced by pandoc is:

```
\documentclass[]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdfauthor={Andrea},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

\author{ag91}
\date{}

\begin{document}

\emph{σ}

\end{document}
```

Thanks a lot!

**Comment by jgm** [comment]
The org reader produces

     [Para [Emph [Str "\963"]]]

Some alternatives would be

     [Para [Math InlineMath "\\sigma"]]

(which is what the org reader does with `$\sigma$`) or

     [Para [Rawinline (Format "latex") "\\sigma"]]

I'm not very knowledgeable about org mode, so let me pass
this on to @tarleb (who wrote the org mode reader) for
comment.

**Comment by jgm** [comment]
Btw, the reason you get no output with
--latex-engine=xelatex is probably that the font you're
using doesn't have a glyph for sigma.

You can change that by setting the `mainfont` variable
to a font that has Greek characters.

**Comment by ag91** [comment]
Sorry, I did not read the FAQ carefully (http://pandoc.org/faqs.html).
It works with the following command:
`pandoc -o c.pdf --latex-engine=xelatex -V mainfont='Standard Symbols
L'`
Thanks!

**Comment by tarleb** [comment]
I'll have a look nonetheless, this reader behavior seems a little weird.

**Comment by tarleb** [comment]
I found a spacing bug related to LaTeX-style symbols, fixed in #2900.  Other than that, things look ok.
